src/main/scala/codacy/plugins/test/MetricsTests.scala

```diff
@@ -0,0 +1,91 @@
+package codacy.plugins.test
+
+import com.codacy.analysis.core.tools.MetricsTool
+import com.codacy.plugins.api.Source
+import com.codacy.plugins.metrics.traits
+import com.codacy.plugins.api.metrics.{FileMetrics, LineComplexity}
+import codacy.plugins.test.Utils.exceptionToString
+
+import scala.util.{Failure, Success}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/gh/codacy/codacy-plugins-test/pullRequest?prid=3735299)